src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs

```diff
@@ -127,4 +127,12 @@ internal class SafeSharedAsn1ObjectHandle : SafeInteriorHandle
         {
         }
     }
+
+    internal class SafeSharedAsn1OctetStringHandle: SafeInteriorHandle
```

Comment:
I assume the purpose of this type being different from the one before it is simply type safety and being able to define APIs that take one or the other, having the compiler ensure we're not passing one handle to an API that accepts a different one, etc.?  That's fine, just making sure I understand the rationale.